crates/serpen/tests/fixtures/ast_rewriting/happy_path/main.py

```
@@ -0,0 +1,35 @@
+    # Use utility functions
+    message = format_message("Welcome", user.name)
+    total = calculate_total([10, 20, 30])
+
+    # Print results
+    print(message)
+    print(f"User: {user}")
+    print(f"Role: {user.role.value}")
+    print(f"Database: {db.connection_string}")
+    print(f"Total: {total}")
+
+    return {"user": user.name, "email": user.email, "role": user.role.value, "total": total, "message": message}
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)